renovate.json

```diff
@@ -118,7 +118,7 @@
   "regexManagers": [
     {
       "fileMatch": [
-        "^.github/workflows/continuous-integration.yml$"
+        "^docker-compose.yml$"
       ],
       "matchStrings": [
         "(?<depName>cypress/included):(?<currentValue>[0-9.]+)\n"
```

Comment:
@BacLuc is this regexManager even needed anymore if we have the container specified normally in docker-compose.yml?

Reply:
Renovate should detect images in the docker-compose, this should not be needed anymore.
But the best way to check is to push your changes to devel in your fork, and run:
`docker run --rm -e LOG_LEVEL=debug renovate/renovate --token $TOKEN --include-forks true --pr-hourly-limit 0 --pr-concurrent-limit 0 --binary-source install $YOUR_FORK/ecamp3 | less`
(you need a github token with the following permissions:)
![image](https://user-images.githubusercontent.com/1506818/223085290-b9e5afd1-fd96-4874-b8b7-e64ae3c96490.png)

Reply:
Will do, thanks!

Reply:
Was successful, here the relevant part of the output:
```
{
 "depName": "cypress/included",
 "currentValue": "12.0.0",
 "replaceString": "cypress/included:12.0.0",
 "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
 "datasource": "docker",
 "updates": [
   {
     "bucket": "non-major",
     "newVersion": "12.7.0",
     "newValue": "12.7.0",
     "newMajor": 12,
     "newMinor": 7,
     "updateType": "minor",
     "branchName": "renovate/cypress-included-12.x"
   }
 ],
 "warnings": [],
 "versioning": "docker",
 "registryUrl": "https://index.docker.io",
 "currentVersion": "12.0.0",
 "isSingleVersion": true,
 "fixedVersion": "12.0.0"
},
```